### ISSUE TITLE
Tacotron 2 NaN loss when using pretrained model as starting point

### ISSUE THREAD

**Issue by PedroDKE** [TensorSpeech/TensorFlowTTS#496]
I'm trying to finetune the pretrained Tacotron 2 model on my own (English) dataset. 
I follow the steps explained [here](https://github.com/tensorspeech/TensorFlowTTS/tree/master/examples/tacotron2#finetune-tacotron-2-with-ljspeech-pretrained-on-other-languages).
My new dataset uses a processor similar to the one from the Thorsten dataset, but uses the English cleaners. To get the vocabulary length of my dataset I've added my own lines at [this](https://github.com/tensorspeech/TensorFlowTTS/blob/master/tensorflow_tts/configs/tacotron2.py#L66) point in my local files to point to my custom processor/symbols.

This seems to be working as i get the following output before training:

`2021-02-13 15:21:58,003 (hdf5_format:779) WARNING: Skipping loading of weights for layer encoder due to mismatch in shape ((63, 512) vs (149, 512)).
2021-02-13 15:21:59,306 (train_tacotron2:466) INFO: Successfully loaded pretrained weight from ../pretrained/tacotron2/model-120000.h5.`

But during the first evaluation of the training set i get the following output:
`
INFO: (Step: 150) train_stop_token_loss = nan.
2021-02-13 15:37:59,026 (base_trainer:1014) INFO: (Step: 150) train_mel_loss_before = nan.
2021-02-13 15:37:59,026 (base_trainer:1014) INFO: (Step: 150) train_mel_loss_after = nan.
2021-02-13 15:37:59,027 (base_trainer:1014) INFO: (Step: 150) train_guided_attention_loss = nan.
`
i did not add any argument similar to 'var_train_expr: "embeddings|encoder|decoder" ' as i would like to train all layers.
Any idea what is causing this and how i can fix it? I already disabled mixed precision training and this did not get rid of the problem.

and ass an fyi: training a model from scratch (using the same command,but leaving out the --pretrained args) does work as it should

it is probably the same problem as in #194 but using MFA i dont get accurate durations (some letters are left out, while they shouldnt). My use case is the same as in the issue, were i want to use tacotron2 to extract the durations for my dataset. Do you think that just using a tacotron2 model from scratch will do this with a reasonable accuracy? and how many iterations should i train it for?

**Comment by dathudeptrai** [comment]
@PedroDKE training from scratch is ok, do not need to fine-tune from our checkpoints. You should be training around 60k-80k steps to be able to get a good duration for fastspeech2.

**Comment by PedroDKE** [comment]
After training the tacotron2 model to extract the durations (70k steps on ~5k audio files) i made a little script to plot the MFCC's all extracted features (f0,energy and durations). In here i saw that in some (i'd say roughly 25% of the cases) the durations dont look too good (its already improved over using MFA or a pretrained tacatron2 in my case though). Should i train the tacotron2 model for more steps or would these kind of durations to be fine to train a fastspeech2 model?  I have attached a screenshot of both train/eval losses and two examples where the durations are off. Any tips?

transcript: i am changed, but you must always be my friend.eos
![9-00009-f000107](https://user-images.githubusercontent.com/43861296/108058072-c97de680-7053-11eb-845f-a5023f331a53.png)

transcript: or was it indifferent to results ?eos
![9-00008-f000331](https://user-images.githubusercontent.com/43861296/108058078-cb47aa00-7053-11eb-8de1-1a8f6d9cc1d5.png)

![train](https://user-images.githubusercontent.com/43861296/108061641-e7017f00-7058-11eb-9cb9-9aaf3cef80d1.png)
![eval](https://user-images.githubusercontent.com/43861296/108061648-e8cb4280-7058-11eb-906d-9d1741d39b66.png)
